CONTRIBUTING.md

```
@@ -0,0 +1,147 @@
+git checkout -b feature/my-new-feature
+```
+
+### 2. Make and commit your changes
+
+Write clean, readable code. Add tests for new functionality and update documentation where relevant.
+
+Commits must be **signed**. See the [GitHub docs](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for setup instructions.
+
+Use the following commit message format:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)